src/Apps/Order2/Routes/Checkout/Components/OfferStep/Forms/Order2ExactPriceOfferForm.tsx

```diff
@@ -0,0 +1,270 @@
+import {
+  Radio,
+  Box,
+  Flex,
+  RadioGroup,
+  Spacer,
+  Text,
+  Button,
+  TextArea,
+} from "@artsy/palette"
+import { OfferInput } from "Apps/Order/Components/OfferInput"
+import { appendCurrencySymbol } from "Apps/Order/Utils/currencyUtils"
+import type { OfferFormComponentProps } from "Apps/Order2/Routes/Checkout/Components/OfferStep/types"
+import type { Order2ExactPriceOfferForm_order$key } from "__generated__/Order2ExactPriceOfferForm_order.graphql"
+import { useEffect, useState } from "react"
+import { graphql, useFragment } from "react-relay"
+
+interface PriceOption {
+  key: string
+  value: number
+  description: string
+}
+
+interface Order2ExactPriceOfferFormProps extends OfferFormComponentProps {
+  order: Order2ExactPriceOfferForm_order$key
+}
+
+export const Order2ExactPriceOfferForm: React.FC<
+  Order2ExactPriceOfferFormProps
+> = ({
+  order,
+  offerValue,
+  offerNoteValue,
+  formIsDirty,
+  isSubmittingOffer,
+  onOfferValueChange,
+  onOfferNoteChange,
+  onOfferInputFocus,
+  onContinueButtonPressed,
+}) => {
+  const orderData = useFragment(FRAGMENT, order)
+  const artwork = orderData.lineItems?.[0]?.artwork
```

Comment:
How would this work if we are trying to make offer on editioned artwork with multiple edition sets?